chart/templates/configmap.yaml

```
@@ -14,6 +14,14 @@ data:
       metricsAddr: "127.0.0.1:8080"
       leaderElection: true
       metricsSink: {{ .Values.controller.metricsSink | quote }}
+      notifiers:
+        common:
+          clustername: {{ .Values.controller.notifiers.common.clusterName | quote }}
```

```suggestion
          clusterName: {{ .Values.controller.notifiers.common.clusterName | quote }}
```

It can sound as a detail but better keeping the same naming everywhere